src/lib/template/templatizer.html

```diff
@@ -168,6 +170,13 @@
       }
     },
 
+    __setPropertyImpl: function(property, value, fromAbove, node) {
+      if (node && node.__hideTemplateChildren__ && property == 'textContent') {
+        property = '__polymerTextContent__';
+      }
+      this.__setPropertyOrig(property, value, fromAbove, node);
+    },
+
```

Comment:
Not thrilled with patching this in the hot-path here; `applyEffectValue` would be slightly better but its not as easy to patch...